lib/node/limits.js

```diff
@@ -40,12 +42,16 @@ function limitInputRows(node, input, context, limit, callback) {
     });
 }
 
-function limitInputRowsAndAvgGroupedRows(node, input, column, context, lims, callback) {
+function limitInputRowsAndAvgGroupedRows(node, input, categoriesData, context, lims, callback) {
```

Comment:
What about calling this `target` instead of `categoriesData`?

Reply:
The categories could be in the source (which is how it was before the refactor) so could be a bit missleading

Reply:
What about `categoryTarget`? I don't like the `data` part because it seems like it's already containing the required data.

Reply:
Hum, sound good to me. Changing it :)